docs/guide/cli.md

```diff
@@ -52,5 +52,14 @@
 | `--environment <env>` | Runner environment (default: `node`) |
 | `--passWithNoTests` | Pass when no tests found |
 | `--allowOnly` | Allow tests and suites that are marked as `only` (default: false in CI, true otherwise) |
-| `--changed [since]` | Run tests that are affected by the changed files (default: false)
+| `--changed [since]` | Run tests that are affected by the changed files (default: false). See [docs](#changed)
 | `-h, --help` | Display available CLI options |
+
+### changed
+
+- **Type**: `boolean | string`
+- **Default**: false
+
+Run tests only against changed files. If no value is provided, it will run tests against uncomitted changes (includes staged and unstaged).
```

Comment:
```suggestion
Run tests only against changed files. If no value is provided, it will run tests against uncommitted changes (including staged and unstaged).
```